modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_users_admin/js/AccountUsersAdminManagementToolbarPropsTransformer.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {
+	createPortletURL,
+	navigate,
+	openSelectionModal,
+	postForm,
+} from 'frontend-js-web';
+
+const updateAccountUsers = (portletNamespace, url) => {
+	const form = document.getElementById(`${portletNamespace}fm`);
+
+	if (!form) {
+		return;
+	}
+
+	postForm(form, {
```

Comment:
Just like @markocikos pointed in https://github.com/liferay-lima/liferay-portal/pull/1507#discussion_r577635010

> See https://github.com/liferay-frontend/liferay-portal/pull/759#discussion_r574501571
> 
> If it's just one line in if block, postForm, it's hard to justify an early return.

Reply:
Sounds good. I will make that change.

I read that comment differently than you as if it was a 1 line or just a few lines of code, we would wrap the `if` check around the entire code block, but if it was more than that, we would return early.  I prefer wrapping the entire code block since that was our practice before, but I agree with the early return justification.

Thanks for clarifying this!